config/krakend/scripts/uniconfig_rbac.lua

```diff
@@ -0,0 +1,26 @@
+function user_group_auth(request, permited_groups)
+    local method = request:method()
+    local headers_group = request:headers('X-Auth-User-Groups')
+    local headers_role = request:headers('X-Auth-User-Roles')
+
+    local headers_all = headers_group .. "," .. headers_role
+    -- remove white spaces in groups
+    headers_all = headers_all:gsub("%s+", "")
+    permited_groups = permited_groups:gsub("%s+", "")
+
+    -- if request method is different from GET
+    --  then check if user is in permitted groups
+    if method ~= "GET" then
+        for header_group in string.gmatch(headers_all, '([^,]+)') do
+            for permited_group in string.gmatch(permited_groups, '([^,]+)') do
+                if header_group == permited_group then
```

Comment:
I would do "toUpper" or "toLower" to ignore case when comparing groups ... I think that's done in WF proxy and Uniresource